src.ts/non-fungible-token.ts

```diff
@@ -354,8 +354,8 @@ export class NonFungibleToken {
                 name: checkString,
                 symbol: checkString,
                 owner: allowNull(checkAddress),
```

Comment:
suggest that the `owner` should not be part of the`tokenProperties` due to the `owner` is the sender itself.